roofit/roofitcore/inc/RooAbsReal.h

```diff
@@ -463,7 +463,39 @@ class RooAbsReal : public RooAbsArg {
 
   //--------------------------------------------------------------------
 
- protected:
+  // Experimental AD feature functions
+  protected:
+  /// the variable that stores the result expression for the object.
+  std::string _adResult = "";
```

Comment:
```suggestion
  std::string _adResult = ""; ///<!
```
You have to make this "magic" comment for cling to know that the `_adResult` doesn't take part in the IO. This is how to fix some of the errors in the CI.

Reply:
Thanks!